src/sentry/api/endpoints/project_user_details.py

```diff
@@ -1,11 +1,33 @@
+from rest_framework import status
 from rest_framework.response import Response
 
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.serializers import serialize
+from sentry.auth.superuser import is_active_superuser
 from sentry.models import EventUser
 
 
 class ProjectUserDetailsEndpoint(ProjectEndpoint):
     def get(self, request, project, user_hash):
         euser = EventUser.objects.get(project_id=project.id, hash=user_hash)
         return Response(serialize(euser, request.user))
+
+    def delete(self, request, project, user_hash):
+        """
+        Delete an Event User
+        ````````````````````````````````
+
+        Delete an event's user.
+
+        :pparam string organization_slug: the slug of the organization.
+        :pparam string project_slug: the slug of the project.
+        :pparam string user_hash: the user hash.
+        """
+        if is_active_superuser(request):
+            euser = EventUser.objects.filter(project_id=project.id, hash=user_hash)
+            euser[0].delete()
```

Comment:
Might make sense to use `get` instead of `filter`, then catch `EventUser.DoesNotExist` and return a 404 if the user doesn't exist